### PR TITLE
Add warning icon with color example

### PR DESCRIPTION
**Pull request by arzezak**
Screenshot 2023-09-21 at 7 18 00 PM

### CODE REVIEW

File: app/assets/images/icons/warning-triangle.svg
Comment (tysongach) [inline app/assets/images/icons/warning-triangle.svg:1]: To make sure icons stay consistent, where was this asset sourced from?

Comment (arzezak) [reply app/assets/images/icons/warning-triangle.svg:1]: According to the note on the default IconComponent preview, https://iconoir.com/.

Comment (tysongach) [reply app/assets/images/icons/warning-triangle.svg:1]: Bingo! Thanks.

Comment (arzezak) [reply app/assets/images/icons/warning-triangle.svg:1]: Using it on https://github.com/BuoySoftware/BuoyRails/pull/9494.

**Review by tysongach — APPROVED** [review]

**Pull request merged**